src/plonk/circuit.rs

```diff
@@ -1062,12 +1062,13 @@ impl<F: Field> ConstraintSystem<F> {
     pub fn enable_constant(&mut self, column: Column<Fixed>) {
         if !self.constants.contains(&column) {
             self.constants.push(column);
-            self.enable_equality(column.into());
+            self.enable_equality(column);
         }
     }
 
     /// Enable the ability to enforce equality over cells in this column
-    pub fn enable_equality(&mut self, column: Column<Any>) {
+    pub fn enable_equality<C: Into<Column<Any>>>(&mut self, column: C) {
```

Comment:
This public change OTOH does make sense to me, since we _do_ want the ability to enable any column to participate in equality constraints, and we aren't returning any value that the caller needs to care about.